apps/web/lib/services/chat/prehydration-pin.test.ts

```
@@ -0,0 +1,75 @@
+      value: 0,
+      writable: true,
+      configurable: true,
+    });
+    log.append(scroller);
+    document.body.append(log);
+    return scroller;
+  }
+
+  function runScript() {
+    // The shipped inline script string is the unit under test.
+    (0, eval)(PREHYDRATION_PIN_SCRIPT);
```

Biome (2.5.7)

[error] eval() exposes to security risks and performance issues.

(lint/security/noGlobalEval)